pisek/jobs/parts/task_job.py

```diff
@@ -183,3 +190,25 @@ def _diff_files(self, file_a: TaskPath, file_b: TaskPath) -> str:
             stdout=subprocess.PIPE,
         )
         return diff.stdout.decode("utf-8")
+
+    def _file_content(
+        self,
+        file: TaskPath,
+        force: bool = False,
```

Comment:
Here it's clear what `force` means, but not when it's forwarded from another function.